server/pb_hooks/pages/index.ejs

```diff
@@ -4,4 +4,173 @@
 </div>
 <div class="w-1/1">
   <%- include('specials.ejs') %>
+</div>
+<!-- Coffee Grid -->
+<div class="grid grid-cols-1 md:grid-cols-3">
+  <div class="grid grid-cols-5 col-span-2 row-span-3 pb-6">
+    <h2 class="text-6xl pl-0 col-span-full">Coffee</h2>
+    <p class="col-span-full">Fair Trade · Organic</p>
+    <ul class="col-span-2">
+      <li>Espresso</li>
+      <li>Americano</li>
+      <li>Depth Charge</li>
+      <li>Cappucino</li>
+      <li>Latte</li>
+      <li>Breve</li>
+      <li>Mocha</li>
+      <li>Caramel Latte</li>
+      <li>Miel</li>
+      <li>Vietnamese</li>
+      <li>Chai Tea</li>
+      <li>Cold Press</li>
+      <li>Hot Chocolate / Steamer</li>
+      <li>Cortado</li>
+    </ul>
+    <!-- small prices -->
```

Comment:
Is it too chaotic that the prices are so separate from the drinks?? I basically ended up separating everything by column rather than grouping by what it fits with. This strikes me as annoying to read, but also the clearest path forward with the grid